### ISSUE TITLE
Convert code to Python

### ISSUE THREAD

**Issue by dlej** [dlej/ica_reg#3]
Matlab is too difficult to maintain and isn't any fun. Re-implement code in Python [3] and try taking advantage of nice gradient descent utilities. Try [Tensorflow](https://www.tensorflow.org/) and see if we can take advantage of the auto-differentiation capabilities.

**Comment by dankessler** [comment]
:+1:

**Comment by dlej** [comment]
As of 7723137bf00c087288e677a7d6d2a506be32ec8e, the code in `ica_reg.py` does the same thing as the old MATLAB code, but in Tensorflow.
